backend/inspirehep/mailing/templates/mailing/jobs/deadline_passed/base.html

```diff
@@ -0,0 +1,10 @@
+<p>Dear {{ job.acquisition_source.email }},</p>
+<p>&nbsp;</p>
+<p>The INSPIRE Jobs listing for your position: <a href="{{ host }}/jobs/{{ job.control_number}}">{{ host }}/jobs/{{ job.control_number }}&nbsp;</a>has an expired deadline: {{ job.deadline_date }}.</p>
+<p>Please modify the deadline date or change the job status to closed through the form:<br /><a href="{{ host }}/submissions/jobs/{{ job.control_number }}">{{host}}/submissions/jobs/{{job.control_number}}</a>.</p>
+<p>Note that closed jobs are no longer visible in the search results.</p>
+<p>&nbsp;</p>
+<p>You can find more information on how to update and close a job posting in our <a href="https://blog.inspirehep.net/2019/07/discover-brand-new-inspire-beta-jobs/">blog</a>.&nbsp;</p>
+<p>Generally we do not remove jobs simply because the deadline has expired as deadlines are often extended. <br />The INSPIRE Jobs database receives thousands of hits a day from people in the high energy physics community. Our goal is to make it useful and efficient for people both advertising and seeking positions. We therefore wish to keep our listings as current as possible.</p>
+<p>Your assistance is greatly appreciated.</p>
+<p>Best Wishes,<br /> <br /> The INSPIRE Jobs Team</p>
```

Comment:
Did you copy paste this from somewhere? there are way too many paragraphs (including empty ones).